_posts/2019-04-09-announcing-riff-0-3-0.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: single
+title: "Announcing riff v0.3.0"
+header:
+  overlay_image: /images/frosty-morning.jpg
+excerpt:
+  With updated Knative components, Windows CLI, and a new Buildpacks design for Functions
+permalink: /blog/announcing-riff-0-3-0/
+---
+
+We are happy to announce the release of [riff v0.3.0](https://github.com/projectriff/riff/releases/tag/v0.3.0). Thank you all riff, Buildpacks, and Knative contributors.
+
+The riff CLI can be downloaded from our [releases page](https://github.com/projectriff/riff/releases/tag/v0.3.0) on GitHub. Please follow one of the [getting started](/docs) guides, to create a new cluster on GKE, Minikube, Docker Desktop for Mac, or Docker Desktop for Windows.
+
+## Notable changes
+
+### Cloud Native Buildpacks
+
+- update to [pack v0.1.0](https://github.com/buildpack/pack/releases/tag/v0.1.0)
+- [builder](https://github.com/projectriff/builder/blob/v0.2.0/builder.toml) uses separate buildpack per function invoker 
+- The Java buildpack includes support for Java 11.  Please add a source version of Java 8 or later to your Maven pom or Gradle build.
+
+  #### Maven
+  ```xml
+  <properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
we need to add target.release here

```
        <maven.compiler.target>1.8</maven.compiler.target>
```

Reply:
i was wondering about that - how about the gradle side?

Reply:
nothing additional needed for gradle